node/src/daemon/setup_reporter.rs

```diff
@@ -3440,10 +3440,10 @@ mod tests {
     #[test]
     fn scan_intervals_computed_default_persistent_config_unequal_to_default() {
         let mut scan_intervals = *DEFAULT_SCAN_INTERVALS;
-        scan_intervals.pending_payable_scan_interval = scan_intervals
-            .pending_payable_scan_interval
+        scan_intervals.payable_scan_interval = scan_intervals
+            .payable_scan_interval
             .add(Duration::from_secs(15));
-        scan_intervals.pending_payable_scan_interval = scan_intervals
+        scan_intervals.receivable_scan_interval = scan_intervals
             .receivable_scan_interval
             .sub(Duration::from_secs(33));
```

Comment:
Can you please add another case for `pending_payable_scan_interval`? Maybe you can test the boundary by adding or subtracting 1.